src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
+import { Comment } from '../../types/types';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  list: Comment[];
+  idOfPost: number;
+};
+
+export const CommentList: React.FC<Props> = (props) => {
+  const array = props.list.find(
```

Comment:
Use array `filter` method instead of `find` to get an array of filtered comments matching the postId. 🧐

Reply:
consider renaming `array` to more meaningful value, like `filteredCommentsList`